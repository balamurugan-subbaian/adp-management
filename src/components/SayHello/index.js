import React from 'react';
import './SayHello.scss';

const SayHello = () => (
    <div>
        <h3 className="heading">Hey there...!</h3>
        <p>Welcome to adp management</p>
    </div>
)

export default SayHello;