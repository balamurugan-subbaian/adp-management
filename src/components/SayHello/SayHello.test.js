import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import SayHello from './index';

describe('<SayHello />', () => {
    test('should render with one wrapper div', () => {
        const wrapper = shallow(<SayHello name="testName"/>);
        expect(wrapper.find('div')).toHaveLength(1);
        expect(toJSON(wrapper)).toMatchSnapshot();
    })
})