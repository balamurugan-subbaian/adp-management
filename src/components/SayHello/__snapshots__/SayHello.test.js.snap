// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SayHello /> should render with one wrapper div 1`] = `
<div>
  <h3
    className="heading"
  >
    Hey there...!
  </h3>
  <p>
    Welcome to adp management
  </p>
</div>
`;
